MaxText/configs/base.yml

```diff
@@ -204,6 +210,7 @@ stack_trace_interval_seconds: 600  # Stack trace collection frequency in seconds
 use_iota_embed: False
 # use positional embedding
 use_positional_embedding: False
+trainable_position_size: 0  # enable gpt3 position embedding with a positive trainable_position_size
```

Comment:
Seems like a weird name and description? Is there a reference or anything.

Reply:
If 0 means off, please use -1? I also think this is related to the above entry?

In which case I'd prefer just glomming them together into one variable:
`position_embedding_size` (-1 to disable)

Reply:
Here's info. 
`trainable_position_size` is used only in a  trainable_position_embedding. It represents the `num_embeddings` in trainable_position_embedding like `vocab_size` in `token embedder`.

The range of position_embedding_size should be any positive int.
```
if cfg.use_positional_embedding:
      if cfg.trainable_position_size:
        # trainable positional embedding, same implementation as `Embed`
        y += Embed(
          num_embeddings=cfg.trainable_position_size,
          features=cfg.emb_dim,
          dtype=cfg.dtype,
          embedding_init=nn.initializers.normal(stddev=1.0),
          name='position_embedder',
          config=cfg)(decoder_positions)
      else:
        # untrainable positional embedding
        y = PositionalEmbedding(cfg.base_emb_dim)(y, decoder_positions)
```

Let me know what you think.